lib/matplotlib/_layoutbox.py

```diff
@@ -456,10 +456,11 @@ def layout_from_subplotspec(self, subspec,
         return lb
 
     def __repr__(self):
-        args = (self.name, self.left.value(), self.bottom.value(),
-                self.right.value(), self.top.value())
-        return ('LayoutBox: %25s, (left: %1.3f) (bot: %1.3f) '
-               '(right: %1.3f)  (top: %1.3f) ') % args
+        return (f'LayoutBox: {self.name:25s} '
```

Comment:
There was a comma after name. Did you remove that intentionally?

Reply:
restored the comma